web/libs/datamanager/src/components/Common/SDKButtons.jsx

```diff
@@ -1,12 +1,16 @@
 import { useSDK } from "../../providers/SDKProvider";
-import { Button } from "./Button/Button";
+import { Button } from "@humansignal/ui";
 
 const SDKButton = ({ eventName, ...props }) => {
   const sdk = useSDK();
 
   return sdk.hasHandler(eventName) ? (
     <Button
       {...props}
+      size="small"
```

Comment:
should we allow size to be passed in as an option? seems like we're required to use small here otherwise
```suggestion
      size={props.size ?? "small"}
```

Reply:
good point